build/commands/lib/util.js

```diff
@@ -517,10 +517,27 @@ const util = {
     Log.progressFinish('touch original vector icon files overridden by brave/vector_icons')
   },
 
+  touchOverriddenPolicyScript: () => {
```

Comment:
if you move this to chromium_src and use the same name as the original script this is will not be needed cc @goodov

Reply:
@darkdh you need to create `chromium_src/components/policy/tools/generate_policy_source.py` counterpart with a content like this:
```
import override_utils

@override_utils.override_function(globals())
def _LoadJSONFile(orig_func, json_file):
  json = orig_func(json_file)
  AddBravePolicies(json)
  return json

def AddBravePolicies(...):
  ...
```

and add a patch into `components/policy/tools/generate_policy_source.py`.

use this as example:
https://github.com/brave/brave-core/blob/master/chromium_src/tools/clang/pylib/clang/compile_db.py
https://github.com/brave/brave-core/blob/master/patches/tools-clang-pylib-clang-compile_db.py.patch

Reply:
https://github.com/brave/brave-browser/wiki/Patching-Chromium#patching-py-files

Reply:
That is much better, fixed in a9987caa4c5a673db5f9561a813a205c255f6bd7